templates/title.mustache

```diff
@@ -20,16 +20,16 @@
 				</a>
 			</li>
 			<li>
-			<a class="github-button nav-social" href="https://github.com/canjs/{{name}}"
-				data-count-href="/canjs/{{name}}/stargazers"
-				data-count-api="/repos/canjs/{{name}}#stargazers_count">Star</a>
-	  	</li>
+				<a class="github-button nav-social" href="https://github.com/canjs/{{repoName}}"
+				   data-count-href="/canjs/{{repoName}}/stargazers"
+				   data-count-api="/repos/canjs/{{repoName}}#stargazers_count">Star</a>
+			</li>
 		</ul>
 	{{/if}}
 	{{#with (getClosestWithPackage)}}
 		<ul class="title-links">
 			<!-- <li><a href="#">docco</a></li> -->
-			{{#with (sourceLink .)}}<li><a href="{{.}}">Edit on GitHub</a></li>{{/with}}
+			<li><a href="{{sourceLink (repoName)}}">Edit on GitHub</a></li>
```

Comment:
I did a quick check on if `docMapInfo.getCurrent().src === undefined`  occurs. It does, but apparently only when rendering subsection headers, for example, "behaviors" under the can-connect. That "virtual parent page" doesn't have content so it's just an ordering in the side menu and doesn't render a page where this <a> tag without a link would be visible. 

From what I can tell the removal of the `{{#with}}` shouldn't have any side effects.